.github/actions/json-test/Dockerfile

```diff
@@ -11,14 +11,16 @@ ARG DEBIAN_FRONTEND="noninteractive"
 RUN apt-get update -q \
  && apt-get install -y --no-install-recommends -qq \
     python3 \
-    python3-jsonschema \
+ #   python3-jsonschema \
     python3-simplejson \
     python3-jsonpath-rw \
     python3-pip \
     python3-setuptools \
-    python3-wheel
+    python3-wheel \
+    git
 
-RUN pip3 install jsonschema[format] --upgrade
+#RUN pip3 install jsonschema[format] --upgrade
+RUN pip3 install git+https://github.com/anexia-it/jsonschema@draft2020-12
```

Comment:
In order to fix the setup just make sure pip is updated to the latest version before:
```
RUN pip3 install --upgrade pip
```

I see the format is skipped, this can be achieved by providing an extra argument in the URL:
```
RUN pip3 install git+https://github.com/anexia-it/jsonschema@draft2020-12#egg=jsonschema[format]
```

Reply:
@nezhar Thank you for the hint!